e2e_tests/dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.12
+ARG parent_dir
+# this environment variable specifies where playwright should download browsers and ffmpeg, unused for now
+# ENV PLAYWRIGHT_BROWSERS_PATH=/app/pw-browsers
+WORKDIR /e2e_tests
+
+# seems like poetry doesn't pick up pyproject.toml for some unapparent reason - same with the readme, might break if removed
+COPY pyproject.toml pyproject.toml
+RUN touch README.md
+
+RUN pip install poetry
+RUN poetry install
+RUN poetry run playwright install --with-deps
+
+# runs the e2e visual regression tests for webkit, firefox and chromium
+CMD poetry run pytest test_visual_regression.py --base-url=https://host.docker.internal:5000/ --browser webkit --update-snapshots; \
```

Comment:
You can run across multiple browsers with 1 command, just by adding extra flags:
`--browser webkit --browser firefox --browser chromium`

Reply:
got it, this quite ideal, I previously wrote it down as that because of the comment below, should I remove the comment?
<img width="828" alt="image" src="https://github.com/nationalarchives/da-ayr-beta-webapp/assets/105435961/9313f5ee-12c2-4039-9633-32e68ba38f00">

Reply:
This comment is still needed, it refers to the fact we pass browser_name into here and we will continue to need to do this until that github issue is fixed.

Reply:
Added it back